### PR TITLE
Add cbor access to CCLConfiguration.

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
API to access the CCLConfiguration through CBOR.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11473

### CODE REVIEW

**Review by pascal-brause — APPROVED** [review]

**Review by pascal-brause — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfigurationAccess.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import SwiftCBOR
+
+enum CCLConfigurationAccessError: Error {
```
Comment (pascal-brause) [inline src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfigurationAccess.swift:8]: must be public to access it

**Review by pascal-brause — CHANGES_REQUESTED** [review]
The `CCLConfiguration` must be declared public in the Toolkit, too.

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfigurationAccess.swift (hunk 1 above)
Comment (ArturFriesen) [reply src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfigurationAccess.swift]: Done

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfigurationAccess.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import SwiftCBOR
+
+public enum CCLConfigurationAccessError: Error {
+    case CBOR_DECODING_FAILED(Error?)
+    case JSON_ENCODING_FAILED(Error?)
```
Comment (nickguendling) [inline src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfigurationAccess.swift:10]: Looks like this not used

Comment (ArturFriesen) [reply src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfigurationAccess.swift]: removed

**Review by nickguendling — APPROVED** [review]

**Pull request merged**